server/util.go

```diff
@@ -373,6 +373,11 @@ func GetAppDBBackend(opts types.AppOptions) dbm.BackendType {
 	if len(rv) == 0 {
 		rv = cast.ToString(opts.Get("db-backend"))
 	}
+
+	if rv == "cleveldb" || rv == "badgerdb" || rv == "boltdb" {
```

Comment:
Looks like magic value, maybe we should add a comment saying it was previously supported db that are now unsupported.

Reply:
added, left a todo and open an issue to write a db migration tool, I can pick that up soon